front/src/index.tsx

```diff
@@ -28,6 +28,7 @@ if (sentryDsn) {
       new CaptureConsole({
         levels: ["error"],
       }),
+      // eslint-disable-next-line import/namespace
```

Comment:
can we get rid of this?

Reply:
Until I find out why ESLint or TS is not finding the method even though it's clearly exported, we need to leave it there.